stock_putaway_product_form/models/putaway_strategy.py

```diff
@@ -0,0 +1,19 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class FixedPutAwayStrategy(models.Model):
+    _inherit = "stock.fixed.putaway.strat"
+
+    product_tmpl_id = fields.Many2one(
```

Comment:
Please use a related field instead of computed one

Reply:
Done